lib/std/Build/Fuzz/WebServer.zig

```diff
@@ -456,6 +463,19 @@ fn sendCoverageContext(
 
         prev_entry_points.* = coverage_map.entry_points.items.len;
     }
+
+    if (duration >= std.time.ns_per_s) {
+        const time_s: usize = @intCast(duration / std.time.ns_per_s);
+        const window_size = n_runs -% last_cycle_timed.*;
+        const cycles_per_second = window_size / time_s;
+        last_cycle_timed.* = n_runs;
+
+        try web_socket.writeMessage(std.mem.asBytes(&abi.PeriodicUpdateHeader{
+            .cycles_per_second = cycles_per_second,
```

Comment:
it should be calculated *client side*.

the only additional thing that needs to be sent by the server is:
 * what time the server thinks it currently is
 * what time according to the server fuzzing started

then the client can do the calculation based on what time it is currently and the number of runs.